src/gutenberg2zim/download.py

```diff
@@ -73,12 +72,12 @@ def is_safe(fname):
     # move all extracted files to proper locations
     for zipped_file in zipped_files:
         # skip folders
-        if not Path(zipped_file).ext:
+        if not Path(zipped_file).suffix:
             continue
```

Comment:
Now that we have a great function to skip folders, let's use this (previous code was a bit ... risky)
```suggestion
        if not Path(zipped_file).is_file:
```